### ISSUE TITLE
VGG gradients explode significantly after epoch one

### ISSUE THREAD

**Issue by Nikronic** [Nikronic/CoarseNet#8]
After passing epoch one, suddenly gradients explode about `10^30` times. I addressed a good implemenation EnhanceNet using Tensorflow in loss function at MSE loss definition line.

**Comment by Nikronic** [comment]
Please see commit https://github.com/Nikronic/CoarseNet/commit/3382bd3760f0be9078eb0827dc2141af06f547f5

**Comment by Nikronic** [comment]
Problem solved.
The problem caused when I tried to run the model on my local environment and using the CPU. Everything sounds fine on the GPU based on the test on Google colab.
See commit https://github.com/Nikronic/CoarseNet/commit/b62c4644cbb5584518f1829f30983af0e270e27a .
